epicbox-hyperskill/gradle/checker/template/build.gradle

```diff
@@ -1,20 +1,21 @@
 plugins {
     id 'application'
-    id 'org.jetbrains.kotlin.jvm' version '1.5.30'
+    id 'org.jetbrains.kotlin.jvm' version '2.0.0'
     id 'scala'
 }
 
-sourceCompatibility = 11
+sourceCompatibility = 21
 
 repositories {
     mavenCentral()
+    maven { url "https://packages.jetbrains.team/maven/p/hyperskill-hs-test/maven" }
     maven { url "https://jitpack.io" }
 }
 
 dependencies {
     implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk8'
-    implementation 'org.scala-lang:scala-library:2.13.6'
-    testImplementation 'com.github.hyperskill:hs-test:v10.0.3'
+    implementation 'org.scala-lang:scala-library:2.13.12'
+    testImplementation 'com.github.hyperskill:hs-test:release-SNAPSHOT'
```

Comment:
You need to specify an explicit version. Each change to the libraries must change the image tag, in this case, when you change the tag, the hash will change. Otherwise, due to the fact that the images are cached, it will be very difficult for us to make a quick fix in the library and roll it out to users

Reply:
@meanmail when we publish a library under a specific index, we need to change the library index in all projects on the platform. We published the latest releases under the "release-SNAPSHOT" tag